src/Core/src/Platform/Android/EntryExtensions.cs

```diff
@@ -59,5 +59,13 @@ public static void UpdateIsTextPredictionEnabled(this EditText editText, IEntry
 		{
 			editText.SetInputType(entry);
 		}
+
+		public static void UpdateIsReadOnly(this EditText editText, IEntry entry)
+		{
+			bool isReadOnly = !entry.IsReadOnly;
+
+			editText.FocusableInTouchMode = isReadOnly;
+			editText.Focusable = isReadOnly;
```

Comment:
Just found out this doesn't stop pastes and other types of non-typing inserts.

It looks like `InputType = InputTypes.Null;` helps a bit.